src/modules/hip/kernel/down_mixing.hpp

```diff
@@ -0,0 +1,72 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+
+__global__ void down_mixing_hip_tensor(float *srcPtr,
+                                       uint srcStride,
+                                       float *dstPtr,
+                                       uint dstStride,
+                                       int2 *srcDimsTensor)
+
+{
+    int id_x = hipBlockIdx_x * hipBlockDim_x + hipThreadIdx_x;
+    int id_z = hipBlockIdx_z * hipBlockDim_z + hipThreadIdx_z;
+    int srcLength = srcDimsTensor[id_z].x;
+    int channels = srcDimsTensor[id_z].y;
+
+    if (id_x >= srcLength)
+        return;
+
+    float outVal = 0.0f;
+    uint srcIdx = id_z * srcStride + id_x * channels;
+    int i = 0;
+    int alignedChannels = (channels / 8) * 8;
```

Comment:
This is rounding down. Is that what you want?

Reply:
Yes Rajy
Since we want to do 8 pixel vectorized processing for alignedChannels value